CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [0.40.0] - 2024-03-21
+
+-   Adds Remix example reference.
```

Comment:
no need for a new version, cause this is not actually a SDK change. Just an example app addition.

Reply:
debumped, 0.39 now.

Reply:
You should remove the `## [0.40.0] - 2024-03-21` from the changelog as well. You can instead use `## [unreleased]`